### PR TITLE
Format beta integration

### PR DESCRIPTION
**Pull request by merit-maita**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/41557
fixes : https://github.com/demisto/etc/issues/27635

## Description
skipped docker validation for beta integrations.

## Must have
- [ ] Tests
- [ ] Documentation
- [ ] [VS Code Extension](https://docs.gitlab.com/ee/ci/yaml/#tags)
  - [ ] Functionality implemented in the extension - <u>\<link-to-pr-here\></u>
  - [ ] N/A - Functionality cannot be implemented in the extension because <u>\<add-reason-here\></u>

### CODE REVIEW

**Review by Bargenish — DISMISSED** [review]
Please add a ref to the issue.
Instead of skipping format for beta integrations, let's try and add support to it.
Dont see a reason to skip it.

File: demisto_sdk/commands/format/update_integration.py [hunk 1]
```diff
@@ -41,6 +43,10 @@ def __init__(self,
         self.update_docker = update_docker
         if not from_version and self.data.get("script", {}).get("type") == TYPE_PWSH:
             self.from_version = '5.5.0'
+        self.is_beta = False
+        integration_type = find_type(input)
+        if integration_type:
+            self.is_beta = True if find_type(input).value == 'betaintegration' else False
```
Comment (yaakovi) [inline demisto_sdk/commands/format/update_integration.py:49]:
```suggestion
            self.is_beta = find_type(input).value == 'betaintegration'
```

File: demisto_sdk/commands/format/update_description.py [hunk 2]
```diff
@@ -28,23 +30,40 @@ def __init__(self,
                  update_docker: bool = False,
                  **kwargs):
         super().__init__(input, output, path, from_version, no_validate, verbose=verbose, **kwargs)
+        description_type = input.replace('_description.md', '.yml')
+        self.is_beta = False
+        file_type = find_type(description_type)
+        if file_type:
+            self.is_beta = True if find_type(description_type).value == 'betaintegration' else False
```
Comment (yaakovi) [inline demisto_sdk/commands/format/update_description.py:37]:
```suggestion
            self.is_beta = find_type(description_type).value == 'betaintegration'
```

File: demisto_sdk/commands/format/tests/test_formatting_md_test.py [hunk 3]
```diff
@@ -1,32 +1,13 @@
...
     SOURCE_DESCRIPTION_FORMATTED_CONTRIB_DETAILS,
-    SOURCE_DESCRIPTION_WITH_CONTRIB_DETAILS)
+    SOURCE_DESCRIPTION_FORMATTED_WITH_BETA_DESCRIPTION,
+    SOURCE_DESCRIPTION_WITH_CONTRIB_DETAILS,
+    SOURCE_DESCRIPTION_WITHOUT_BETA_DESCRIPTION)
 
 
 class TestDescriptionFormat:
-    @pytest.fixture(autouse=True)
-    def description_copy(self):
```
Comment (yaakovi) [inline demisto_sdk/commands/format/tests/test_formatting_md_test.py:16]: why did you remove the fixtures?

Comment (merit-maita) [reply demisto_sdk/commands/format/tests/test_formatting_md_test.py:16]: wanted both tests to have the same logic, and i cant use the same fixtures to both of the tests

**Review by yaakovi — APPROVED** [review]
